src/main/java/com/cloudogu/scm/review/pullrequest/dto/ReviewerDto.java

```diff
@@ -0,0 +1,19 @@
+package com.cloudogu.scm.review.pullrequest.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.AllArgsConstructor;
+
+@Getter
+@Setter
+@ToString
```

Comment:
You really need ToString and AllArgConstructor?

Reply:
You're right both are not necessary. But for instantiating I need a default constructor.